### PR TITLE
Feature/kyv 787 swagger documentation addition

### PR DESCRIPTION
**Pull request by synthko**
Added openapi documentation for /refund/instant/{orderId} endpoint.

### CODE REVIEW

File: packages/payment-experience-api/src/openapi.yaml [hunk 1]
```diff
@@ -414,4 +552,50 @@ paths:
                 $ref: '#/components/schemas/Payment'
         '404':
           $ref: '#/components/responses/NotFound'
-
+  /refund/instant/{orderId}:
+    get:
+      description: Create a refund for single order, confirm it and create a payment - refunds all order items.
```
Comment (severikupari1) [inline packages/payment-experience-api/src/openapi.yaml:557]: Create a payment-> create a refund payment

File: packages/payment-experience-api/src/openapi.yaml [hunk 2]
```diff
@@ -414,4 +552,50 @@ paths:
...
+      security:
+        - ApiKey: [ ]
+      parameters:
+        - $ref: '#/components/parameters/OrderId'
+      responses:
+        '200':
+          description: Creates a refund for given order and all of its orderItems, confirms it and creates payment.
```
Comment (severikupari1) [inline packages/payment-experience-api/src/openapi.yaml:564]: Same here, add refund payment

**Review by severikupari1 — APPROVED** [review]
Minor changes

File: packages/payment-experience-api/src/openapi.yaml (hunk 1 above)
Comment (synthko) [reply packages/payment-experience-api/src/openapi.yaml:557]: Good point, I'll update the text.

**Pull request merged**